apps/issuer/serializers_v1.py

```diff
@@ -224,6 +226,44 @@ class BadgeClassExpirationSerializerV1(serializers.Serializer):
                                        choices=BadgeClass.EXPIRES_DURATION_CHOICES)
 
 
+class SuperBadgeBadgeSerializerV1(serializers.ModelSerializer):
+    id = serializers.RelatedField(queryset=BadgeClass.objects.all())
```

Comment:
What ID is this? Why is this query set requesting all badge classes? Probably a comment clarifying this would be helpful